js/sessions.js

```diff
@@ -12,6 +12,16 @@ loadJSON(function (response) {
     createSessionList("Day-1", day1);
     createSessionList("Day-2", day2);
 
+    setTimeout(function () {
+        var hash = window.location.hash;
+        if (hash) {
+            var $sessionId = $('.top' + hash);
+            $sessionId.click();
+            $('html, body').animate({
+                scrollTop: $sessionId.offset().top - 120 //Minus header + shadow
+            }, 300);
+        }
+    }, 100);
```

Comment:
I'm guessing this timeout is so that we load the data and have the id's in place. While this might work most of the time, we might instead callback to this function after the data and DOM are updated.

Reply:
@claytron5000 : Implemented quick fix instead of changing the whole logic. We should be calling this one in callback instead of `setTimeout`. `setTimeout` will work when set of data is smaller.